e3sm_to_cmip/__main__.py

```diff
@@ -3,349 +3,134 @@
-        if self._pool:
-            self._pool.close()
-            self._pool.terminate()
-            self._pool.join()
-# ------------------------------------------------------------------
-
-def parse_argsuments():
-    parser = argparse.ArgumentParser(
-        description='Convert ESM model output into CMIP compatible format',
-        prog='e3sm_to_cmip',
-        usage='%(prog)s [-h]')
-    parser.add_argument(
-        '-v', '--var-list',
-        nargs='+',
-        required=True,
-        metavar='',
-        help='space seperated list of variables to convert from e3sm to cmip. Use \'all\' to convert all variables')
-    parser.add_argument(
-        '-i', '--input',
-        metavar='',
-        required=True,
-        help='path to directory containing e3sm data with single variables per file')
-    parser.add_argument(
-        '-o', '--output',
-        metavar='',
-        help='where to store cmorized output')
-    parser.add_argument(
-        '-u', '--user-input',
-        required=True,
-        metavar='<user_input_json_path>',
-        help='path to user input json file for CMIP6 metadata')
-    parser.add_argument(
-        '-n', '--num-proc',
-        metavar='<nproc>',
-        default=6, type=int,
-        help='optional: number of processes, default = 6')
-    parser.add_argument(
-        '-t', '--tables',
-        required=True,
-        metavar='<tables-path>',
-        help="Path to directory containing CMOR Tables directory")
-    parser.add_argument(
-        '-H', '--handlers',
-        metavar='<handler_path>',
-        default=None,
-        help='path to cmor handlers directory, default = ./cmor_handlers')
-    parser.add_argument(
-        '-N', '--proc-vars',
-        action='store_true',
-        help='Set the number of process to the number of variables')
-    parser.add_argument(
-        '--version',
-        help='print the version number and exit',
-        action='version',
-        version='%(prog)s 0.0.2')
-    parser.add_argument(
-        '--debug',
-        help='Set output level to debug',
-        action='store_true')
-    try:
-        _args = sys.argv[1:]
-    except:
-        parser.print_help()
-        sys.exit(1)
-    else:
-        _args = parser.parse_args(_args)
-    return _args
+import numpy as np
+np.warnings.filterwarnings('ignore')
 
 
 def main():
+
     # parse the command line arguments
-    _args = parse_argsuments()
+    _args = parse_argsuments().__dict__
+
+    var_list = [x.strip(',') for x in _args.get('var_list')]
+    input_path = _args.get('input_path')
+    output_path = _args.get('output_path')
+    tables_path = _args.get('tables_path')
+    user_metadata = _args.get('user_metadata')
+    no_metadata = _args['no_metadata'] if _args.get('no_metadata') else False
+    nproc = _args['num_proc'] if _args.get('num_proc') else 6
+    serial = _args['serial'] if _args.get('serial') else False
+    mode = _args['mode'] if _args.get('mode') else 'atm'
```

Comment:
There doesn't seem to be any corresponding argument in the arg parser, so the mode is always `atm`.